messages/en.json

```diff
@@ -134,7 +134,7 @@
 	"faq-agents-user": "includes all people who view a page. Editors, anonymous editors, and our readers.",
 	"faq-agents-spider": "includes search engine \"<a href='https://en.wikipedia.org/wiki/Web_crawler'>web crawlers</a>\" like Google that read pages for the purposes of improving search results. Pages can receive significant views from web crawlers, which is why by default the tool shows data only from users.",
 	"faq-agents-bot": "includes any other automated programs that may scrape pages for other purposes.",
-	"faq-chart-type-title": "I don't like this kind of chart",
+	"faq-chart-type-title": "How can the change chart type be changed?",
```

Comment:
Good idea to make it not-so-negative. `How can I change the chart type?` is how I would word it, or at least `How can the chart type be changed?` (removing "change"). Letting you change it so you can get credit for the commit :)